addons/website_event_track/controllers/event.py

```diff
@@ -25,17 +7,14 @@
 
 import pytz
 
-import openerp
-import openerp.tools
 from openerp.addons.web import http
 from openerp.addons.web.http import request
+from openerp.tools import html_escape
 
 class website_event(http.Controller):
     @http.route(['''/event/<model("event.event"):event>/track/<model("event.track", "[('event_id','=',event[0])]"):track>'''], type='http', auth="public", website=True)
     def event_track_view(self, event, track, **post):
-        track_obj = request.registry.get('event.track')
-        track = track_obj.browse(request.cr, openerp.SUPERUSER_ID, track.id, context=request.context)
-        values = { 'track': track, 'event': track.event_id, 'main_object': track }
+        values = { 'track': track, 'event': track.event_id }
```

Comment:
Why removing the browsing with SUPERUSER_ID ? Are you sure it was not necessary ?